docs/devel/developer-guides/vagrant.md

```diff
@@ -34,45 +34,74 @@ Documentation for other releases can be found at
 
 ## Getting started with Vagrant
 
-Running kubernetes with Vagrant (and VirtualBox) is an easy way to run/test/develop on your local machine (Linux, Mac OS X).
+Running kubernetes with Vagrant (and VirtualBox) is an easy way to
```

Comment:
I asked instead that we state clearly its not tested on a per PR basis, and ask users to open an issue for support if there is a regression.  I do use this environment about once a week.